payments/providers/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 from .bambora_payform import BamboraPayformProvider  # noqa
 # imported here so that we can refer to it like 'payments.providers.BamboraPayformProvider'
+from .cpu_ceepos import CPUCeeposProvider # noqa
```

Comment:
If this is imported just for exporting it via the package name space (i.e. not used), then it should be put to the `__all__` list. There shouldn't be any need for the `noqa` comment.

Reply:
@suutari-ai In this case the `__all__` list would be like this, right?
```python
__all__ = [
    'BamboraPayformProvider',
    'CPUCeeposProvider',
    'PaymentProvider',
]
```

Reply:
I think also load_provider_config and get_payment_provider should be in the list.

But now that I think it through: Does it even make sense to add those subpackages to the namespace of the `providers` package directly? It's just forcing all the providers to be imported even if only one (or zero) of them is specified in the settings. This might even need extra unneeded requirements to be installed even if they're unused. There's at least e.g. Stripe which has its own Python package and it wouldn't be nice to force installation of that package if that provider is not even used.

Reply:
At this point I would leave this as it is, if you think it's fine.